Mathlib/RingTheory/RingHom/StandardSmooth.lean

```diff
@@ -31,14 +31,25 @@ namespace RingHom
 variable {R : Type u} {S : Type v} [CommRing R] [CommRing S]
 
 /-- A ring homomorphism `R →+* S` is standard smooth if `S` is standard smooth as `R`-algebra. -/
+@[algebraize RingHom.IsStandardSmooth.toAlgebra]
```

Comment:
Does this mean that `algebraize` does not check that this name exists already?
Should this promise be checked in CI?